test/lib/ufe/testAttribute.py

```diff
@@ -1311,40 +1311,41 @@ def testObservationWithFineGrainedNotifications(self):
         ball35Obs.assertNotificationCount(self)
         globalObs.assertNotificationCount(self, numValue = 6)
 
-        # Undo-ing the modification which created the USD specs is a little
-        # different in USD, but from Ufe we should just still see one notification.
+        # The spec creation done on the first setValue was filtered so it
+        # appeared only as a single value change. Symetrically, the unto
+        # of the spec creation should be filtered as well.
         cmds.undo()
-        ball34Obs.assertNotificationCount(self, numRemoved = 1, numValue = 6)
+        ball34Obs.assertNotificationCount(self, numValue = 7)
```

Comment:
Makes more sense that if numAdded ended up being zero, then numRemoved should follow.